Makefile

```diff
@@ -114,6 +110,15 @@ check_os_env:
 		echo 'You need to set the APPLET_PUBLIC_KEY variable to a valid authentication key path'; \
 		exit 1; \
 	fi
+	@if [ "${APPLET_PATH}" == "" ]; then \
+		echo 'You need to set the APPLET_PATH variable to a valid path for the applet elf and signature files'; \
```

Comment:
We should probably give a hint either here in the error message, or in the README, that this is typically the full path to the .../assets directory in your clone of the armored-witness-applet repo.

Reply:
See if https://github.com/transparency-dev/armored-witness-os/pull/8/commits/dc35fe04db3f5c71c4aab09185b2cf4c1906fac1 resolves this.